tests/backup_restore_test.go

```diff
@@ -67,6 +68,7 @@ func TestServer_BackupAndRestore(t *testing.T) {
 			t.Fatalf("error backing up: %s, hostAddress: %s", err.Error(), hostAddress)
 		}
 	}()
+	return // TEMP
```

Comment:
Remove?

Reply:
This test seems to pass when I remove these temporary statements.